File: Basic Coding Problems/01 Fundamentals/008 Write a program to enter temperature in Celsius and convert it into Fahrenheit/C++/convert_celsius_to_fahrenheit.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    float celsius, fahrenheit;

    cout << "Enter temperature in Celsius: ";
    cin >> celsius;

    fahrenheit = (celsius * 9 / 5) + 32;

    cout << "Temperature in Fahrenheit: " << fixed << setprecision(2) << fahrenheit << endl;

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/008 Write a program to enter temperature in Celsius and convert it into Fahrenheit/Java/convert_celsius_to_fahrenheit.java
import java.util.Scanner;

public class convert_celsius_to_fahrenheit {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter temperature in Celsius: ");
        float celsius = scanner.nextFloat();

        float fahrenheit = (celsius * 9 / 5) + 32;

        System.out.printf("Temperature in Fahrenheit: %.2f\n", fahrenheit);
    }
}